metashare/context_processors.py

```diff
@@ -2,5 +2,8 @@
 
 
 def env(request):
-    GLOBALS = {"SENTRY_DSN": settings.SENTRY_DSN}
+    GLOBALS = {
+        "SENTRY_DSN": settings.SENTRY_DSN,
+        "DEVHUB_USERNAME": settings.DEVHUB_USERNAME,
```

Comment:
The frontend doesn't actually need to know what the username is, right? So it would be nice to only expose a boolean indicating whether it's set, rather than the actual username.

Reply:
@davisagli @wlonk I'll note that we're still exposing this value to the front-end via `user.sf_username`. If we don't want to do that, we can omit it with a custom serializer if using the global devhub.

Reply:
Can make this happen if we decide to!